domainlab/compos/pcr/request.py

```diff
@@ -6,6 +6,12 @@ class RequestVAEBuilderCHW():
     def __init__(self, i_c, i_h, i_w, args):
         pass
 
+class RequestVAEBuilderNN():
+    """creates request when input does not come from command-line (args) but from test_exp file"""
+    @store_args
+    def __init__(self, net_class_d, net_x, net_class_y, i_c, i_h, i_w):
+        """net_class_d, net_x and net_class_y are neural networks defined by the user"""
+        pass
```

Comment:
there is no need to use pass here, as long as there is documentation, it is fine, using pass will confuse the reader